types/dockerode/index.d.ts

```diff
@@ -475,6 +475,7 @@ declare namespace Dockerode {
         Name: string;
         RestartCount: number;
         Driver: string;
+        Platform?: string;
```

Comment:
Why is it optional?

Reply:
Since this property wasn't specified before and old API doesn't have this property, made it optional.

Reply:
@ravipal IMO it doesn't matter that it wasn't present in the past (nobody would ever instantiate their own `ContainerInspectInfo` object, only Dockerode would ever do that, and Dockerode doesn't care about what is defined here).

However, the weird thing is that `Platform` property is _not_ present in the [API specification](https://docs.docker.com/engine/api/v1.40/#operation/ContainerInspect), even though it is present in the result. @thegecko given that maybe it is best to leave it optional?

Reply:
@bwateratmsft it is specified in the response schema. Since the old version didn't specify this property it is better to leave it as optional.

Reply:
Where is it in the response schema?

That the old versions of `@types/dockerode` lacked this property doesn't matter; it won't break any code because nobody would instantiate a `ContainerInspectInfo`.

Reply:
In the same link that you provided expand the 200 no error.
![image](https://user-images.githubusercontent.com/1775825/81593594-89772e00-9374-11ea-81b1-54a2bf7be93a.png)

Reply:
Ah, I was looking at the response sample on the right, which lacked it. In that case though I agree with @thegecko, it doesn't need to be optional.

Reply:
How about handling the old api version which doesn't seem to have this property.

Reply:
Do you mean old version of `@types/dockerode` or old version of the Docker Engine HTTP API?

Reply:
latest version of types/dockerode interacting with old version of Docker Engine.

Reply:
`@types/dockerode` doesn't interact with Docker Engine, `dockerode` does that--and `dockerode` doesn't use `@types/dockerode`. The only scenario where making this property required would cause issues is if somebody is instantiating a `ContainerInspectInfo` object, but there's really no reason to ever do that (except for tests, maybe).

Reply:
As it's a response object,  adding the new field shouldn't break older implementations, so it doesn't need to be optional.

Reply:
As long as it doesn't break while interacting with older docker engine, I am fine. removed the optional.

Reply:
Thanks!